sdk/src/test/java/io/radar/sdk/RadarTest.kt

```diff
@@ -226,6 +226,7 @@ class RadarTest {
         assertNotNull(address?.borough)
         assertNotNull(address?.county)
         assertNotNull(address?.neighborhood)
+        assertNotNull(address?.street)
         assertNotNull(address?.number)
```

Comment:
check `address?.addressLabel`?

Reply:
Omitting since it's not always present (this test is used for both geocode & autocomplete)